server.js

```diff
@@ -0,0 +1,138 @@
+import express from "express";
+import bodyParser from "body-parser";
+import mongoose, { ObjectId } from "mongoose";
+const app = express();
+const PORT = 3001;
+const connectionString =
+  "mongodb+srv://Prohar:PEA16april2005@cluster0.bosgtkc.mongodb.net/?retryWrites=true&w=majority";
+
+const URL = {
+  GREETINGS: "/api/greetings",
+  NOTES: "/api/notes",
+};
+
+app.use(express.json());
+
+// DB CONNECTION
+mongoose
+  .connect(connectionString, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+  .then(() => {
+    console.log("Connected to MongoDB");
+  })
+  .catch((error) => {
+    console.error("Error connecting to MongoDB:", error.message);
+  });
+
+// SCHEMA
+const schema = {
+  id: Number,
+  title: String,
+  content: String,
+  createAt: Date,
+  updateAt: Date,
+};
+const mongooseModel = mongoose.model("NEWCOL", schema);
+
+// TASK1
+app.get(URL.GREETINGS, function (req, res) {
+  try {
+    const name = req.query.name;
+    if (name) {
+      res.send(`hello ${name}`);
+    } else {
+      throw new Error("there is no value");
+    }
+  } catch (error) {
+    console.log(error);
+  }
+});
```

Comment:
Let's distinguish 2 logical parts from the list of tasks: greetings and notes.
My suggestion here to create new folders: `src/routes` and `src/controllers`
Describe inside src/routes/index.js:
```
routers.use(ROUTES.greetings, greetingsRouter);
routers.use(ROUTES.notes, notesRouter);
```
And inside src/routes/greetingsRouter.js:
```
app.get(ROUTES.default, GreetingsController.sendGreeting)
```
GreetingsController located here: src/controllers/greetingsController.js:
```
export const sendGreeting = (req, res, next) => {
  try {
    const name = req.query.name;
    if (name) {
      res.send(`hello ${name}`);
    } else {
      throw new Error("there is no value");
    }
  } catch (error) {
    console.log(error);
  }
};
```
Going back to the server.js, it will look like: 
`app.use(ROUTES.default, routes);`